includes/Story_Renderer/HTML.php

```diff
@@ -264,6 +267,55 @@ protected function print_analytics( $content ) {
 		return str_replace( '</amp-story>', $output . '</amp-story>', $content );
 	}
 
+	/**
+	 * Print amp-story-bookend before closing `</amp-story>`.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @param string $content String to replace.
+	 *
+	 * @return string
+	 */
+	protected function print_bookend( $content ) {
+		$share_providers = [
```

Comment:
In the future, it would be nice to add these options as a setting in the dashboard, so you can config them yourself.